### PR TITLE
pyqt5_sip: init at 12.9.1

### PR DESCRIPTION
**Pull request by doronbehar**
###### Things done

I hope it's OK I put @SuperSandro2000 as a maintainer of this new package, since it's a dependency of `pyqt5` which you are listed as maintainer.

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
@svanderburg not me.

File: pkgs/development/python-modules/pyqt/sip.nix
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pyqt/sip.nix:17]: Can we do an pythonImportsCheck instead?

File: pkgs/development/python-modules/pyqt/sip.nix
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pyqt/sip.nix:7]:
```suggestion
  pname = "pyqt5-sip";
```
normalised it should look like this.

**Review by flokli — APPROVED** [review]
Thanks! I built this, and now the gui works :-)

File: pkgs/development/python-modules/pyqt/sip.nix
Comment (flokli) [reply pkgs/development/python-modules/pyqt/sip.nix:17]: The package outputs contain (except metadata) only a single .so binary.

**Comment by flokli** [comment]
> @svanderburg not me.

pyqt5 is pretty much broken without it. It requires it for operation, as it's using this to bind to the qt libs: https://www.riverbankcomputing.com/software/sip/intro

File: pkgs/development/python-modules/pyqt/sip.nix
Comment (flokli) [reply pkgs/development/python-modules/pyqt/sip.nix:17]: I assume its functionality can only be checked by interacting with these bindings /from pyqt5/ - and the `pythonImportsCheck` there didn't surface it :-/

**Comment by doronbehar** [comment]
> @svanderburg not me.

Why should they maintain this? I don't see any relation between other packages they maintain to this one.

File: pkgs/development/python-modules/pyqt/sip.nix
Comment (doronbehar) [reply pkgs/development/python-modules/pyqt/sip.nix:17]: `PyQt5.sip` seems to be relevant.

File: pkgs/top-level/python-packages.nix
Comment (doronbehar) [reply pkgs/top-level/python-packages.nix:7361]: > can you put this after `pyqt5`?

Done.

File: pkgs/development/python-modules/pyqt/sip.nix
Comment (doronbehar) [reply pkgs/development/python-modules/pyqt/sip.nix:7]: > normalised it should look like this.

Done.

**Comment by flokli** [comment]
> Why should they maintain this? I don't see any relation between other packages they maintain to this one.

Ah, I confused sander with SuperSandro2000. Sorry!

**Review by flokli — APPROVED** [review]
LGTM.

**Comment by doronbehar** [comment]
> > Why should they maintain this? I don't see any relation between other packages they maintain to this one.
> 
> Ah, seems we both confused SuperSandro2000 with sander, who's maintaining pyqt5. Sorry!

Another maintainer with a different Github login then their maintainer attribute name..

**Comment by doronbehar** [comment]
Will merge when ofborg is green.

**Pull request merged**
